mprof.py

```diff
@@ -402,13 +402,28 @@ def plot_file(filename, index=0, timestamps=True, children=True, options=None):
 
     all_colors = ("c", "y", "g", "r", "b")
     mem_line_colors = ("k", "b", "r", "g", "c", "y", "m")
+
+    show_trend_slope = options is not None and hasattr(options, 'slope') and options.slope is True
+
     mem_line_label = time.strftime("%d / %m / %Y - start at %H:%M:%S",
                                    time.localtime(global_start)) \
                      + ".{0:03d}".format(int(round(math.modf(global_start)[0] * 1000)))
 
+    mem_trend = None
+    if show_trend_slope:
+        # Compute trend line
+        mem_trend = np.polyfit(t, mem, 1)
+
+        # Append slope to label
+        mem_line_label = mem_line_label + " slope {0:.5f}".format(mem_trend[0])
+
     pl.plot(t, mem, "+-" + mem_line_colors[index % len(mem_line_colors)],
             label=mem_line_label)
 
+    if show_trend_slope:
+        # Plot the trend line
+        pl.plot(t, t*mem_trend[0] + mem_trend[1], "--", linewidth=0.5, color="#00e3d8")
```

Comment:
why so thin? It's barely visible in my screen. I would use linewidth=2

Reply:
I've set it that thin because the line itself is "illustrative", having some visual feedback is nice but think that the actual value of this feature is the numeric slope in the label.
That said, I don't have any issue making it width 2